src/plugin/cesium/tiles/cesium3dtileslayer.js

```diff
@@ -134,6 +164,14 @@ plugin.cesium.tiles.Layer.prototype.setOpacity = function(value) {
   }
 };
 
+/**
+ * @param {string} errorMsg The message of the error
+ * @protected
+ */
+plugin.cesium.tiles.Layer.prototype.setTokenError_ = function(errorMsg) {
+  this.tokenError_ = errorMsg;
```

Comment:
You might want to call `this.updateError();` here, instead of after each `setTokenError_` call. Whenever `this.tokenError_` changes we'll likely want to update the error on the layer.

If you want to prevent potential for duplicate updates when the error didn't actually change, you could wrap this block in `if (this.tokenError_ !== errorMsg)`.

Reply:
One other note - if the error changed and is not null/empty, we may also want to fire an alert so the user sees the message prominently in the UI. The icon change is subtle and may be missed.